### PR TITLE
Add User data layer

### PR DESCRIPTION
**Pull request by DavidArt**

### CODE REVIEW

**Comment by sabaubogdan** [comment]
seems ok :)

**Review by sabaubogdan — APPROVED** [review]
Seems ok

File: src/main/java/xyz/vegaone/easytracking/repo/UserRepo.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package xyz.vegaone.easytracking.repo;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import xyz.vegaone.easytracking.domain.UserEntity;
+
+@Repository
+public interface UserRepo extends JpaRepository<UserEntity, Long>{
```
Comment (sabaudragos) [inline src/main/java/xyz/vegaone/easytracking/repo/UserRepo.java:8]: As a general note: do you use CTRL+ALT+L to format code in java classes.

**Review by sabaudragos — APPROVED** [review]

File: src/main/java/xyz/vegaone/easytracking/repo/UserRepo.java (hunk 1 above)
Comment (DavidArt) [reply src/main/java/xyz/vegaone/easytracking/repo/UserRepo.java:8]: nope

**Pull request merged**